lib/modules/datasource/python-version/readme.md

```diff
@@ -0,0 +1,30 @@
+This datasource returns Python releases from the [python.org API](https://www.python.org/api/v2/downloads/release/).
+
+It also fetches deprecated versions from [Endoflife Date Datasource](/modules/datasource/endoflife-date/)
+
+Because Renovate depends on [containerbase/python-prebuild](https://github.com/containerbase/python-prebuild/releases) it will also fetch Github API for releases there.
```

Comment:
```suggestion
Because Renovate depends on [`containerbase/python-prebuild`](https://github.com/containerbase/python-prebuild/releases) it will also fetch releases from the GitHub API.
```